sdk/keyvault/keyvault-keys/src/keysModels.ts

```diff
@@ -107,7 +107,7 @@ export interface DeletedKey {
   /**
    * @member {string} [value] The key value.
    */
-  keyMaterial?: JsonWebKey;
+  key?: JsonWebKey;
```

Comment:
`id`, `name`, `keyType`, and `keyOperations` should also be lifted up to `DeletedKey` (since it is a `KeyVaultKey`).